models/Document/Editable.php

```
@@ -491,7 +491,7 @@ public function __toString()
                 // the __toString method isn't allowed to throw exceptions
```

We could just remove the try/catch, since `__toString` *is* allwoed to throw exceptions since PHP 7.4: https://wiki.php.net/rfc/tostring_exceptions

---

The question is whether this would be a BC break. We could throw the exception only in debug mode instead - that would be definitely safe. WDYT?